projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-tab-edit-modal/dossier-management-widget-tab-edit-modal.html

```diff
@@ -0,0 +1,68 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<cds-modal
+  *ngIf="{disabled: disabled$ | async} as obs"
+  valtimoCdsModal
+  [open]="open$ | async"
+  showFooter="true"
+  title="{{ 'widgetTabManagement.tab.editWidgetTab' | translate }}"
+  class="edit-widget-tab-modal"
+>
+  <cds-modal-header [showCloseButton]="true" (closeSelect)="closeModal()">
+    <h3 cdsModalHeaderHeading>{{ 'widgetTabManagement.tab.editWidgetTab' | translate }}</h3>
+  </cds-modal-header>
+
+  <section cdsModalContent>
+    <ng-container *ngTemplateOutlet="editWidgetTabFormTemplate;context:{obs: obs}"></ng-container>
+  </section>
+
+  <cds-modal-footer>
+    <ng-container *ngTemplateOutlet="editWidgetTabButtons;context:{obs: obs}"></ng-container>
+  </cds-modal-footer>
+</cds-modal>
+
+<ng-template #editWidgetTabFormTemplate let-obs="obs">
+  <form *ngIf="editWidgetTabForm" [formGroup]="editWidgetTabForm" class="widget-tab-modal__form">
+    <div class="form__element">
```

Comment:
This div should not be necessary, add this class directly to the cds-label element